src/components/Toast/CreateToast.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * @Author: zhaohui
+ * @Date: 2021-05-17 14:50:33
+ * @LastEditTime: 2021-05-18 15:06:01
+ * @LastEditors: zhaohui
+ * @Description:
+ * @FilePath: /vant-react/src/components/Toast/CreateToast.tsx
+ */
+import ReactDom from 'react-dom';
+import React from 'react';
+import ToastContainer from './ToastContainer';
+import { ToastItemProps, ToastProps } from './types';
+
+const createToast = () => {
+  const div = document.createElement('div');
+  document.body.appendChild(div);
+  const toast = ReactDom.render(<ToastContainer />, div);
+  let defaultProps: ToastProps = {
```

Comment:
![image](https://user-images.githubusercontent.com/13799221/118972841-00a7b080-b93f-11eb-8050-fd133f0274ec.png)
Sometimes multiple toasts can overlap, can we improve the logic here to prevent 2 toasts from showing at once?

Reply:
I will fix it.